alts/src/main/java/io/grpc/alts/AltsAuthContext.java

```diff
@@ -105,4 +112,13 @@ public RpcProtocolVersions getPeerRpcVersions() {
   public Map<String, String> getPeerAttributes() {
     return context.getPeerAttributesMap();
   }
+
+  /**
+   * Generates a gRPC channel security.
+   *
+   * @return the generated gRPC channel security.
+   */
+  public Security toChannelSecurity() {
```

Comment:
This looks fine to mark as `@Internal`. Although we generally don't do that quite like this. We've developed a pattern where we only expose methods from Internal-prefixed classes, and those classes are a separate bazel target and are excluded from documentation.

For that process, this method should be package-private and you'd create a new class named `InternalAltsAuthContext` and it'd have a method `public Security toChannelSecurity(AltsAuthContext ctx) { return ctx.toChannelSecurity(); }`.

Although don't do that yet. Some of my other comments bring up other issues that will impact this.